src/xdocs/anttask.xml.vm

```diff
@@ -91,6 +91,7 @@
 
     <section name="Parameters">
 
+      <div class="wrapper">
       <table summary="parameters">
```

Comment:
would it be easier to always make the table itself a wrapper if it is part of the a section?